### PR TITLE
Find all cases

### LINKED ISSUE #2: Finding standard words when spelled in upper case
According to [3.4.2 Finding definition names](http://forth-standard.org/standard/usage#subsection.3.4.2):

> A system shall be capable of finding the definition names defined by this standard when they are spelled with upper-case letters.

However:

```forth
here . cr
3087 
ok
HERE . CR
? HUH
```

### PR DESCRIPTION
**Pull request by frno7**
Resolves #2:

```forth
shForth ready.
HERE . CR
3087 
ok
: fOo 42 . Cr ;
ok
fOO
42 
ok
```

### CODE REVIEW

File: memory.sh [hunk 1]
```diff
@@ -10,6 +10,18 @@ peek() {
   eval "echo \$cell_$1"
 }
 
+lowercase() {
+  if test 65 -le "$1" -a $1 -le 90; then # 'A' <= $1 <= 'Z'
```
Comment (zeldin) [inline memory.sh]: Inconsistent use of quotes around $1 here.

File: interpreter.sh [hunk 2]
```diff
@@ -10,11 +10,11 @@ builtin() {
 
 identifier() {
   addr="$1"
-  n=`peek "$addr"`
+  n=`peek_lowercase "$addr"`
```
Comment (zeldin) [inline interpreter.sh]: No.  n is the length of the counted-string, not a graphic.

File: memory.sh [hunk 3]
```diff
@@ -10,6 +10,18 @@ peek() {
...
+    echo `"$expr" "$1" + 32` # $1 + 'a' - 'A'
+  else
+    echo "$1"
+  fi
+}
+
+peek_lowercase() {
```
Comment (zeldin) [inline memory.sh]: This convenience function seems oddly specific.  I'd prefer to just have indentifier() call lowercase() on a temporary variable.

**Review by zeldin — CHANGES_REQUESTED** [review]

**Comment by frno7** [comment]
OK!

**Pull request merged**